### DISCUSSION TITLE
Issues with Phoenix Code 3.8 build 3.8.8.

### DISCUSSION THREAD

**Discussion by arachnidus** [phcode-dev/phoenix#1743] [General]
I am using Release 3.8 build 3.8.8. under Windows11 pro and do have the following issues:

- Phoenix Code crash when I move it to another screen.
- Several plugins failed loading
Don't have the issues with Brackets

**Comment by abose** [comment]
Thanks for reporting the issue, i need a bit more issue before I can address this:
1. Phoenix Code crash when I move it to another screen
   1. Did this happen after you upgraded from an earlier version installed? 
   2. Does it always happen
3. Can you give a list of plugins that failed, did it failed to start or what is the exact problems being faced?

**Comment by arachnidus** [comment]
Hi Arun,

 

Phoenix Code is crashing (no event report in Windows eventviewer) always when I move the application to another screen. I believe it started after the upgrade.

 

Installation of plugins failed, message is “The Extension encountered an error while starting up.

Failed plugin list:

AutoSave every edit 1.1.6

Brackets CSS Class Code hint (root path ok) 0.3.1

Outline List 1.6.0

HTMLEscaper 1.0.2

HTML Format 1.0.2

HTML Tag Autocompleter for Brackets 1.0.0

JSHint Configurator 1.1.6

JSLint Configurator 1.1.13

 

Probably more, 

 

Kind regards,

 

Hans Stad

**Reply by abose** [reply]
I'm unable to repro this with my pc. What is the graphics card/processor. 

1. Have u tried turing it off and on again btw, maybe that may fix it. 
2. Uninstall phcode, and choose delete application data during uninstall. This will lead to losing the editor state live open project lists or extensions installed. Install again and see if that helps.

If both didn't work, it may be a deeper problem to investigate

**Comment by arachnidus** [comment]
Hi Arun,

 

I turn my PC off when I am done with work.

Reinstalling don’t help. Application is still crashing. Are there any debug logs. I turned debug logging on but can’t find the logs

 

My processor and Graphics Card are both AMD, AMD Ryzen 7 5800X 8-Core and AMD Radeon RX 6600. Both up to date by drivers and software. 

 

Kind regards

 

Hans

**Reply by abose** [reply]
Im still unable to recreate this issue, this might be a toigh one to crack without more error reports.

**Comment by arachnidus** [comment]
Which report do you need and how can I debug it with any windows tool.

 

I did some research with processmon and noticed that Phoenix Code did a proper shutdown according to the program

**Reply by abose** [reply]
Does it happen after a restart? I suspeacted a stray process, thats why asked if the system was restarted. Else just try ending all phoenix code process from task manager and try once. 

Else we will need to create custom debug builds for further investigation.

**Comment by arachnidus** [comment]
I shutdown my PC after done with my work.

 

I have done some test:

Windows is running but startup programs are not all started. I can move Phoenix Code around my screens. When I wait till all the startup programs has started Phoenix Code is closing down when I move it to another of my three screens.

 

Those are my startup programs:

**Comment by arachnidus** [comment]
Hi,

 

In the attachment are two log files created by processmon. Maybe it is if any help.

 

It is not a crash but a proper shutdown, shutdown take place even when I move the window on one screen.

 

Kind regards

 

Hans

**Reply by abose** [reply]
The attachments are not visible if you reply via mail. Please upload directly in the gihub discussion with drag an drop.

**Comment by arachnidus** [comment]
[phoenix code shutdown.zip](https://github.com/user-attachments/files/16175798/phoenix.code.shutdown.zip)
is it a csv and a xml file with the same data

**Comment by arachnidus** [comment]
done

**Comment by abose** [comment]
I am unable to recreate this issue and without that it will be very hard to fix this. Waithin on more reports to rule out if this is a single machine specific issue. If you have another windows machine lying around, can you see if its an issue in that too?

**Comment by arachnidus** [comment]
I have tested this in a VM with Windows 11 running on the same hardware and I was able to move around my three screens without problem.

 

Earlier on I wrote that the issue started to appear after all my startup programs are running. There is one thing I can do to help you is to disable all the startup programs and start them one by one till the issue appears. Then we know which program is interfering with Phoenix Code

 

Brackets is working fine but that is a 32-bit program and not a 64-bit like Phoenix-Code

**Reply by abose** [reply]
Thanks for checking it out. If you find anything more, please update in this issue.

**Comment by arachnidus** [comment]
I found the program giving the problem with Phoenix Code. It is named felix and is drawing a walking cat on the screen. 

 

What I believe is that Phoenx Code is closing down because it is not doing a proper refresh of the video memory while moving the window over the screen.

 

Do you like to receive the program for testing, you can’t download it any more

**Reply by abose** [reply]
We work with tauri for the native renderers. Due to internal security policy of tauri, we are only allowed to download and and install executables from public links and with valid code signatures to debug. So we will not be able to register issues with them with the unsupported software. Please do reopen if this happens with any other currently supported software.

**Reply by abose** [reply]
Closing this issue as a one off problem with unsupported software. Please feel free to reopen if you need any more help.

Thanks for helping narrow down the problem and working with us on the same 😊
